tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -160,5 +161,27 @@ IList<string> ReadFile (string path)
 				return Array.Empty<string> ();
 			return File.ReadAllLines (path).Where (line => !line.StartsWith ("#", StringComparison.Ordinal) && line != string.Empty).ToList ();
 		}
+
+		readonly string [] ignoreList = {
+			"ResourceManager",
+			"Culture",
+		};
+
+		[Test]
+		public void UpdatedResources ()
+		{
+			var resxPath = Path.Combine (Directory.GetCurrentDirectory (), "../../../msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx");
```

Comment:
This depends on the current directory, which the test shouldn't really do.

You can use this code to get somewhere relative to the root path of the repository:

```suggestion
			var resxPath = Path.Combine (Configuration.RootPath, "msbuild", "Xamarin.Localization.MSBuild", "MSBStrings.resx");
```

Reply:
OOh nice I like that haha thanks!